doc/guide/cockpit-guide.xml

```diff
@@ -53,6 +53,7 @@
 
   <part id="development">
     <title>Developer Guide</title>
+    <subtitle>See also the documentation in <ulink url="https://github.com/cockpit-project/cockpit/tree/main/doc">the .md files directly in the doc folder of the cockpit source</ulink> and <ulink url="https://github.com/cockpit-project/cockpit/blob/main/HACKING.md">the HACKING.md file in the source</ulink>. These are currently not rendered here, but only the content of the folder doc/guide in the source is rendered here. Help is welcome in converting this documentation from Docbook XML to Markdown and merging these.</subtitle>
```

Comment:
Squeezing this into the subtitle is too much of an abuse:

![image](https://github.com/cockpit-project/cockpit/assets/200109/47d898a8-0b04-44c1-be7f-43e40718352a)

How about moving that into a new sub-file and including it here like the others?

Also, HACKING.md doesn't really belong here. This developer guide is about the cockpit API, i.e. how to use it to create your own pages. HACKING.md is about developing on cockpit itself. I don't mind much mentioning it here somewhere, but this is too prominent and distracting.

tree/main/doc is more useful, in particular `protocol.md`, but also the other files.